lib/maropost/api.rb

```diff
@@ -54,13 +54,23 @@ def update(contact)
       end
 
       def change_email(old_email, new_email)
-        contact = find(old_email)
+        contact = nil
+        old_email_contact = find(old_email)
```

Comment:
you wound not need to do this if we leverage the contact object instead but that I think will be a bigger change